debian/enterprise-control-center/include/etc/confluent/docker/admin.properties.template

```diff
@@ -1,4 +1,14 @@
-{% set security_props = env_to_props('CONTROL_CENTER_STREAMS_PRODUCER_', '') -%}
+{% set producer_security_props = env_to_props('CONTROL_CENTER_STREAMS_PRODUCER_', '') -%}
+{% set security_props = env_to_props('CONTROL_CENTER_STREAMS_', '') -%}
```

Comment:
I think you can do the same with this  `{% set kafka_props = env_to_props('CONTROL_CENTER_STREAMS_', '', exclude= producer_security_props) -%}`